gulpfile.js

```diff
@@ -680,6 +704,9 @@ function parseExtensionFlags() {
     const extensionsMessage = green('⤷ Use ') +
         cyan('--extensions=amp-foo,amp-bar ') +
         green('to choose which extensions to build.');
+    const exampleExtensionsMessage = green('⤷ Use ') +
+        cyan('--extensions=examples/foo.amp.html ') +
+        green('to build extensions from example AMP(s).');
```

Comment:
It seems a little unintuitive to overload the `--extensions` flag across files and extension names. Would it be too cumbersome to use a different flag for this? Say, `--extensions_in_files`?

Reply:
Sure I can do that. FWIW it's already overloaded with a non-extension keyword `minimal_set`, and an `.amp.html` fits the mental model because it's specifying another set. I originally implemented it as a separate argument like you suggest, but I thought there'd be an ergonomic gain from specifying all the extensions you want in one flow.